src/commands/whoami/mod.rs

```diff
@@ -1,20 +1,53 @@
+use crate::http;
 use crate::settings::global_user::GlobalUser;
-use crate::terminal::{emoji, message};
+use crate::terminal::emoji;
+
+use cloudflare::endpoints::account;
+use cloudflare::endpoints::account::Account;
+use cloudflare::framework::apiclient::ApiClient;
+use cloudflare::framework::HttpApiClientConfig;
+
+use prettytable::{Cell, Row, Table};
 
 pub fn whoami(user: &GlobalUser) -> Result<(), failure::Error> {
     // If using email + API key for auth, simply prints out email from config file.
-    let email: String = match user {
-        GlobalUser::GlobalKeyAuth { email, .. } => email.to_string(),
-        GlobalUser::TokenAuth { .. } => failure::bail!(
-            "At the moment, Wrangler cannot get user information for users using API tokens"
-        ),
+    let auth: String = match user {
+        GlobalUser::GlobalKeyAuth { email, .. } => {
+            format!("a Global API Key, associated with the email '{}'", email,)
+        }
+        GlobalUser::TokenAuth { .. } => format!("an API Token"),
     };
 
-    let msg = format!(
-        "{} You are logged in with the email '{}'.",
-        emoji::WAVING,
-        email
-    );
-    message::info(&msg);
+    println!("\n{} You are logged in with {}.\n", emoji::WAVING, auth,);
```

Comment:
why not use message? it's fine but i've been seeing this a lot lately, maybe our message module is not serving us well.

Reply:
ur not wrong. as said in air- the table would be offset if i used messaged because it would print INFO.